generators/getMetric.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const fs = require('fs')
+const objects = require('./getMetricFunctions.js')
+const fsPromises = fs.promises
+const fsExtra = require('fs-extra')
+
+async function generateGetMetric () {
+  return new Promise((resolve, reject) => {
+    try {
+      let generatedDoc = ''
+      for (const object of objects.data) {
+        generatedDoc += allTimeBoundTemplates(object)
+      }
+      let file = path.join(`${__dirname}`, `../src/getMetric.js`)
+      if (fs.existsSync(file)) {
+        fsExtra.removeSync(file)
+      }
+      resolve(fsPromises.writeFile(file, generatedDoc))
+    } catch (error) {
```

Comment:
You can replace the try-catch block by using the promise like `promise.then(...).catch(...)`